source/deployment/bulk-loading-data-format.rst

```diff
@@ -1122,3 +1124,52 @@ Fields of the DirectPost object
       <td align="center" valign="middle">No</td>
     </tr>
   </table>
+
+Emoji object
+-----------
+
+If present, Emoji objects must occur after all User objects.
+
+Example Emoji object
+~~~~~~~~~~~~~~~~~~~
+
+For clarity, the object is shown using regular JSON formatting, but in the data file it cannot be spread across several lines. It must be all on one line.
```

Comment:
I didn't understand this at first but realised it was referring to the example below...  
Suggest >
"For clarity, the example object below is shown using regular JSON formatting.  When used in the data file, it cannot be formatted across several lines and should all be on one line."